tex/chProgramming.tex

```diff
@@ -594,22 +594,24 @@ \section{Data types, first examples}
 \[ 0 + x \times (S\ 0) \]
 is a well formed expression in the language $\{0,S, +, \times\}$ of
 Peano arithmetic. In these examples, however, the usual axioms that
-formalize the respective arithmetic confer a distinctive status to
+give a meaning to the respective arithmetic confer a distinctive status to
 the symbols $\top$ and $\bot$ for booleans, and to the symbols $0$ and
 $S$ for natural numbers. More precisely, any variable-free boolean expression
 is equal to either $\top$ or $\bot$ modulo these axioms, and any
 variable-free expression in Peano arithmetic is equal either to $0$ or
-to an iterated application of the symbol $S$ to $0$. In both
+to an iterated application of the symbol $S$ to $0$.  In both
 cases, the other symbols in the signature represent functions, whose
-computational content is prescribed by the axioms.
+computational content is prescribed by the axioms.  In what follows,
+we shall call the distinctive symbols \emph{constructors}.  So the constructors
+in boolean arithmetic are \(\top\) and \(\bot\) and the constructors of
```

Comment:
Suggesting "in boolean arithmetic" -> "of boolean arithmetic" (see the end of this line).